R/reactives.R

```diff
@@ -357,7 +357,9 @@ Observable <- R6Class(
         stop("Can't make a reactive expression from a function that takes one ",
              "or more parameters; only functions without parameters can be ",
              "reactive.")
-      .func <<- func
+
+      .func <<- wrapFunctionLabel(func, paste("reactive", label),
```

Comment:
That's what I had before, but it looks quite unintuitive due to the file/line number not matching up. The srcref of the reactive name (`count`) is pointing to somewhere deep in Shiny. We can't drop that one from the stack trace either though; if a reactive expr caches an error and re-raises it when called later, then you need that `count` on the stack as a clue for which reactive raised the error.